.github/workflows/turborepo-release-step-2.yml

```diff
@@ -68,6 +72,16 @@ jobs:
       - uses: ./.github/actions/setup-go
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - run: apt-get install -y build-essential musl-tools clang llvm gcc-aarch64-linux-gnu binutils-aarch64-linux-gnu
```

Comment:
It might be worth at some point figuring out which of these are actually necessary. I kinda just dumped a bunch in here to see what worked.